.eslintrc

```diff
@@ -23,13 +23,13 @@
     "no-mixed-requires": [0],
     "no-underscore-dangle": [0],
     "no-bitwise": 2,
-    "camelcase": 1,
+    "camelcase": ["error", { "allow": ["^UNSAFE_"] }],
```

Comment:
Yeah, these are the better way to go about this, good call.

Reply:
👍